src/components/SuggestedAction/SuggestedAction.tsx

```diff
@@ -29,20 +32,30 @@ export interface SuggestedActionProps {
   hideCtas?: boolean;
   /* any children components; optional */
   children?: React.ReactNode;
+  /* these next three props are actions for the hamburger menu, all the arguments are specified in izakaya */
+  /* dismisses the suggestedaction */
+  dismissAction: () => void;
+  /* lets the contractor view the workflow details */
+  viewWorkflow: () => void;
```

Comment:
ugh, this is a broader question again and mostly just a note for later because this is a pretty contextual component; just don't love being super specific about type of user, etc